baron.jl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env julia
+using JuMP
+using ArgParse
+
+using MINLPLib_jump
```

Comment:
Using this as a package is inconsistent with the other scripts which use the "include" method of importing the models from a file directly.  I have a slight preference for "include" approach.